src/WordPress/WordPressClientAPI.php

```diff
@@ -15,10 +15,10 @@ class WordPressClientAPI extends Client
     public function getZoneTag($zone_name)
     {
         $request = new Request('GET', 'zones/', array('name' => $zone_name), array());
-        $response = self::callAPI($request);
+        $response = $this->callAPI($request);
```

Comment:
That's why I changed from `self::` to `$this->`.  Since `public function callAPI(Request $request)` is not static `$this->` should be correct if I'm not missing something.